pkg/kubelet/images/image_gc_manager_test.go

```diff
@@ -824,14 +824,14 @@ func TestGarbageCollectImageTooOld(t *testing.T) {
 
 	// First GC round should not GC remaining image, as it was used too recently.
 	assert := assert.New(t)
-	images, err = manager.freeOldImages(ctx, images, fakeClock.Now())
+	images, err = manager.freeOldImages(ctx, images, fakeClock.Now(), fakeClock.Now(), "")
```

Comment:
This code confuses me a bit.

The last parameter is also a duration so what does "" compile down to for duration?